harness/testAtomics.js

```diff
@@ -44,6 +39,11 @@ function testWithAtomicsInBoundsIndices(f) {
     var good_indices = [
         (view) => 0/-1, // -0
         (view) => '-0',
+        (view) => undefined, // 0
+        (view) => Number.NaN, // 0
```

Comment:
the regular global `NaN` should do the trick as well, but it's not necessary to do change this.